pages/common/nextclade.md

```diff
@@ -0,0 +1,28 @@
+# nextclade
+
+> Bioinformatics tool for virus genome alignment, clade assignment and qc checks.
+> More information: <https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html>.
+
+- Run Nextclade to create TSV report, auto-downloading the latest [d]ataset:
+
+`nextclade run -d {{dataset-name}} {{path/to/fasta}} -t {{path/to/output-tsv}}`
```

Comment:
```suggestion
`nextclade run -d {{dataset_name}} {{path/to/fasta}} -t {{path/to/output_tsv}}`
```

Use long options if they exist and `_` instead of `-` in placeholders.